src/pages/admin/notifications/index.tsx

```diff
@@ -3,33 +3,34 @@ import MyHead from "@/components/MyHead";
 import NotificationTable from "@/components/NotificationTable";
 import Link from "next/link";
 
+type KindType = ["application", "approval_pending", "unapplied"];
+
+const application: KindType[0] = "application";
+const approvalPending: KindType[1] = "approval_pending";
+const unapplied: KindType[2] = "unapplied";
+
 const rows = [
```

Comment:
NotificationTable から RowType をインポートして型注釈する方がスッキリするかなと思います。
そうすれば上のコードは消せますね。

```suggestion
const rows: RowType[] = [
```

Reply:
https://github.com/tatata05/next-app/pull/10/commits/23cd25629055406fc65063aa4e5e1af1ff20bfe7
修正しました！
型定義もimportできたんですね！